support/Pipelines/Bbh/EccentricityControl.py

```diff
@@ -19,29 +19,61 @@
 logger = logging.getLogger(__name__)
 
 
-def ecc_control_fit(
+def eccentricity_control(
     h5_file,
-    subfile_name_aha,
-    subfile_name_ahb,
-    angular_velocity_from_xcts=None,
-    expansion_from_xcts=None,
-    tmin=500,  # make more dynamic later
+    id_input_file_path,
+    subfile_name_aha="ApparentHorizons/ControlSystemAhA_Centers.dat",
+    subfile_name_ahb="ApparentHorizons/ControlSystemAhB_Centers.dat",
+    tmin=500,
     tmax=None,  # use all available data
-    output=None,
-    id_input_file_path=None,  # reads from Inspiral.yaml
+    output=None,  # reads from Inspiral.yaml
 ):
+    """Eccentricity reduction post inspiral.
+
+    This function can be called after the inspiral has run (see the 'Next'
+    section of the Inspiral.yam file).
+
+    This function does the following:
+
+    - Reads orbital parameters from the Inspiral.yaml input file. These
```

Comment:
"Reads initial orbital parameters from the 'id_input_file_path'." You can delete the second sentence.